bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -486,7 +486,7 @@ impl SlidingSyncListBuilder {
 
     pub fn build(self: Arc<Self>) -> Result<Arc<SlidingSyncList>, ClientError> {
```

Comment:
```suggestion
    pub fn build(self: Arc<Self>) -> Arc<SlidingSyncList> {
```

The `Result` is no more required I guess.

Reply:
Good catch, thanks!